api/src/certification/complementary-certification/application/complementary-certification-route.js

```diff
@@ -0,0 +1,33 @@
+import { complementaryCertificationController } from './complementary-certification-controller.js';
+import { securityPreHandlers } from '../../../../lib/application/security-pre-handlers.js';
+
+const register = async function (server) {
+  server.route([
+    {
+      method: 'GET',
+      path: '/api/admin/complementary-certifications/{id}/target-profiles',
+      config: {
```

Comment:
Il manque la définition des retours possibles ainsi que du format des payload

```suggestion
      response: {
          failAction: 'ignore',
          status: {
            200: <TODO_voir_code_pour_exemple>,
            400: erreurDoc,
            401: erreurDoc,
            403: erreurDoc,
          },
        },
        plugins: {
          'hapi-swagger': {
            produces: ['application/json'],
          },
        },
      config: {
```

Reply:
Je ne fait ici que déplacer une route qui existe déjà, ça sors de l'objectif de la PR, je propose de le faire dans un autre ticket

Reply:
d'ailleurs on peut observer que ce n'est pas du tout un usage courant dans les routes Pix, si on cherche à l'introduire ça mériterait un sujet d'aprem tech pour l'inclure dans un maximum de nos routes